cpp/tests/io/orc_chunked_reader_test.cu

```diff
@@ -1338,16 +1338,16 @@ TEST_F(OrcChunkedReaderInputLimitTest, ReadWithRowSelection)
   do {
     auto chunk = reader.read_chunk();
     // Each output chunk should have either exactly 50k rows, or num_rows_to_read % 50k.
-    EXPECT_TRUE(chunk.tbl->num_rows() == 50000 ||
-                chunk.tbl->num_rows() == num_rows_to_read % 50000);
+    // EXPECT_TRUE(chunk.tbl->num_rows() == 50000 ||
```

Comment:
Let's add a comment above all of these why they have been disabled for future reference or remove them altogether?